src/setup_gmail.py

```diff
@@ -65,7 +65,7 @@ def get_key(self):
             return None
 
     def is_patch(self) -> bool:
-        if re.match(r'\[.+ (\d+)/(\d+)\] .+', self.subject):
+        if re.match(r'\[.+\] .+', self.subject):
```

Comment:
Hmm, this can have some false positives.

E.g. `[GIT PULL]`, see https://lore.kernel.org/lkml/20201106173823.48AD72151B@mail.kernel.org/
Maybe we're fine with matching more since it'll just fail to apply and this way we avoid false negatives.

E.g. We could also just match against `'r\[PATCH.*\] .*'`.
This would exclude "[RFC]" and "[RFC PATCH]" (and any other variants thereof).
If we're fine with not including RFCs (I also see someone using "RFC PATCH"

Reply:
Good point, maybe we could reuse some of the patch parser code to check if it is a patch? Like this function: https://github.com/google/lkml-gerrit-bridge/blob/cf15a12740d0e54b1c335f9efdbebc4331e2b01f/src/patch_parser.py#L620 ?

Maybe we can create a lightweight version of the patch. Or maybe even just preparse it. That might go beyond the scope of your change here @WLIU0628.

@dlatypov, what do you think?

Reply:
Hmm.
Having a preparsed `Optional[T]` in `Message` could be a smart way of doing things.

We'd still need to do some regex matching on the subject to grab the patch index in a series so I'm fine with it being left as-is for now in this PR.

We have issue #15 filed now, which we can use to track that further work.

Reply:
SGTM